evaluation/intro/demo-spell.sh

```diff
@@ -1,10 +1,14 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
Why is this change necessary?

Reply:
It's not necessary, just want to make sure this example runs with `sh` (and it does)